docs/schema.md

```diff
@@ -607,6 +614,66 @@ The known reference `type` values are:
 - `PACKAGE`: A web page for the affected package itself.
 - `WEB`: A web page of some unspecified kind.
 
+## credits fields
+
+```json
+{
+	"credits": [ {
+		"name": string,
+		"contact": [ {
+			"type": string,
+			"value": string
+		} ],
+	} ]
+}
+```
+
+The `credits` field is a JSON array providing a way to give credit where credit
+is due related to the discovery, confirmation, patch, or other events in the life
+cycle of a vulnerability.
+
+Each of the objects in the `credits` array must contain at minimum a `name` field
+specifying the name of the individual or entity being credited, using whatever
+notation they prefer.
+
+Each `credits` entry can optionally include a `contact` array. Each entry in the
+`contact` list is an object that indicates the `type` of contact info being
+described and a `value` based on the type. These are described in more detail below.
+
+### credits[].name field
+
+`credits[].name` should specify the name, label, or other identifier of the
+individual or entity being credited, using whatever notation the creditor prefers.
+For instance, this could contain a real name like `Chris Bloom`, an Internet handle
+like `chrisbloom7`, an entity name like `GitHub`, or something else. This field is
+required for each `credits` entry.
+
+### credits[].contact[] field
+
+Each `credits[].contact[]` entry is a JSON object that defines the type of contact
+being described. Providing contact entries is optional, but when present each
+entry must provide both a `type` and `value` property.
+
+#### credits[].contact[].type field
+
+The `credits[].contact[].type` property must be one of those defined below, which
+allows processing systems to interpret and display contact information dynamically
+as necessary and appropriate.
+
+| Contact Type | Value Description |
+| --------- | ----------- |
+| `EMAIL` | A valid, fully qualified, plain-text email adddress at which the credited can be reached. |
+| `GITHUB` | A valid, plain-text GitHub username for the credited. |
+| `TWITTER` | A valid, plain-text Twitter username, with or without the leading `@` symbol, for the credited. |
+| `URL` | A fully-qualified plain-text URL |
+| `OTHER` | A plain-text value that is not represented by any other available type and is self-evident in its usage as a means of contacting the credited individual or entity. |
```

Comment:
I would prefer we omit "OTHER", given that we have a "URL" field. Having a free form "OTHER" field where anything goes isn't very actionable for consumers. Ideally we'd want every field in this schema to be well formed and easily parseable (apart from the English vulnerability description text).